candidate list.txt

```diff
@@ -0,0 +1 @@
+Bruno Cardi Rocky
```

Comment:
Please do not use spaces in file names.
Camel case convention might be useful here.